grails-app/views/order/_filters.gsp

```diff
@@ -22,12 +22,12 @@
             </div>
             <div class="filter-list-item">
                 <label>${warehouse.message(code: 'order.status.label')}</label>
-                <g:select id="status"
-                          name="status"
-                          from="${org.pih.warehouse.order.OrderStatus.list()}"
+                <g:select id="displayStatus"
+                          name="displayStatus"
+                          from="['PENDING','PLACED','SHIPPED','PARTIALLY_RECEIVED','RECEIVED','CANCELED']"
```

Comment:
Perhaps it would be better to add a displayStatusList in OrderStatus as the list from this line